components/Project/index.js

```diff
@@ -115,8 +117,12 @@ export default function Project({ project, favorites, onToggleFavorite }) {
       </ProjectInfoBox>
 
       <ButtonSection>
-        <DeleteButton onClick={handleDelete}>Delete</DeleteButton>
-        <StyledLink href={`${project._id}/edit`}>Edit</StyledLink>
+        {session && session.user.email === project.author && (
+          <DeleteButton onClick={handleDelete}>Delete</DeleteButton>
+        )}
+        {session && session.user.email === project.author && (
+          <StyledLink href={`${project._id}/edit`}>Edit</StyledLink>
+        )}
```

Comment:
instead of checking the user twice you can just do it once: 
```suggestion
        {session && session.user.email === project.author && ( <>
          <DeleteButton onClick={handleDelete}>Delete</DeleteButton>
          <StyledLink href={`${project._id}/edit`}>Edit</StyledLink>
        </>)}
```